src/VirtualAccount.php

```diff
@@ -19,15 +19,15 @@ public function all($options = array())
         return parent::all($options);
     }
 
-    public function close()
+    public function close($id)
     {
-        $relativeUrl = $this->getEntityUrl() . $this->id;
+        $relativeUrl = $this->getEntityUrl() . $id . '/close';
 
         $data = array(
```

Comment:
can this be removed if not being used now?

Reply:
removed